Bottom: 4e534ca94a3c4de9c795c0fe23a1d5d8f6820ccf
Top:    a082027bfd9422fd94a36b36ed2d431da9f27681
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2012-10-10 13:31:46 -0400

From a2f49dc662181764987eecd2b11d503f191c255f Mon Sep 17 00:00:00 2001

Subject: Add a reset function


---

diff --git a/arch/arm/mach-lpc31xx/board-ea313x.c b/arch/arm/mach-lpc31xx/board-ea313x.c
index b7864a4..0e537ec 100644
--- a/arch/arm/mach-lpc31xx/board-ea313x.c
+++ b/arch/arm/mach-lpc31xx/board-ea313x.c
@@ -47,6 +47,7 @@
 #include <mach/gpio.h>
 #include <mach/i2c.h>
 #include <mach/board.h>
+#include <mach/system.h>
 
 static struct lpc313x_mci_irq_data irq_data = {
 	.irq = IRQ_SDMMC_CD,
@@ -735,6 +736,7 @@ MACHINE_START(EA313X, "NXP EA313X")
 	.init_irq	= lpc313x_init_irq,
 	.timer		= &lpc313x_timer,
 	.init_machine	= ea313x_init,
+	.restart	= arch_reset,
 MACHINE_END
 #endif
